nifi-mock/src/main/java/org/apache/nifi/util/verifier/ConditionsBuilder.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.util.verifier;
+
+import java.util.ArrayList;
+
+public class ConditionsBuilder {
+
+    protected final ArrayList<Condition> conditions = new ArrayList<>();
+
+    public ConditionsBuilder(Condition condition) {
+        conditions.add(condition);
+    }
+
+    static public ConditionsBuilder attributeEqual(String name, String value) {
+        return new ConditionsBuilder(new AttributeEqual(name,value));
+    }
+
+    static public ConditionsBuilder contentEqual(String content) {
+        return new ConditionsBuilder(new ContentEqual(content));
+    }
+
+    public ConditionsBuilder andAttributeEqual(String name, String value) {
+        conditions.add(new AttributeEqual(name,value));
+        return this;
+    }
+
+    public ConditionsBuilder andContentEqual(String content) {
+        conditions.add(new ContentEqual(content));
+        return this;
+    }
+
+    public ArrayList<Condition> getConditions() {
+        return conditions;
+    }
+
```

Comment:
You are right.
But we are moving to Java 8 and Predicate will be used instead of ConditionsBuilder.
So this code is not relevant anymore.

Thanks
toivo